### PR TITLE
LPD-20200 Batch Engine Headless API Exporting NotificationTemplate Email Throws an exception

### PR DESCRIPTION
**Pull request by 4lejandrito**
https://liferay.atlassian.net/browse/LPD-20200.

### CODE REVIEW

File: modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/writer/ObjectWriterFactory.java [hunk 1]
```diff
@@ -43,7 +43,7 @@ public static ObjectWriter getObjectWriter(List<String> includeFieldNames) {
 		{
 			disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
 			enable(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY);
-			enable(SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS);
+			configure(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY, true);
```
Comment (ccorreagg) [inline modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/writer/ObjectWriterFactory.java:46]: Is this necessary based on the previous line of code?

**Comment by 4lejandrito** [comment]
ci:test:sf

**Review by GaborKomaromi — APPROVED** [review]

**Comment by ccorreagg** [comment]
ci:forward

**Comment by ccorreagg** [comment]
ci:forward

**Comment by ccorreagg** [comment]
ci:forward

**Comment by ccorreagg** [comment]
ci:forward:force

**Pull request closed without merging**